src/generators/server-side-rendering/visitors/Element.js

```diff
@@ -11,6 +11,10 @@ export default {
 		let openingTag = `<${node.name}`;
 
 		node.attributes.forEach( attribute => {
+			if ( attribute.type === 'Spread' ) {
+				openingTag += ` \${Object.keys( root.${attribute.name} ).map( prop => \`\${prop}="\${root.${attribute.name}[prop]}"\` ).join( ' ' )}`;
```

Comment:
I think this needs some kind of escaping. But then again, the normal attribute code below has no escaping either.